src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -424,6 +424,18 @@
                     </Button>
                 </StackPanel>
             </Border>
+            <Border x:Name="TracePointer"
+                    Width="16"
+                    Height="18"
+                    Margin="50,50,0,0"
```

Comment:
>50,50 [](start = 28, length = 5)

For some reason, David set the initial pointer to be at 50, 50.  Following along here.